### ISSUE TITLE
how to train my own dataset?

### ISSUE THREAD

**Issue by zhangzonghong** [nuclearboy95/Anomaly-Detection-OCGAN-tensorflow#1]

**Comment by MaryAnnZ** [comment]
You have to implement your own data loading method in dataset.py. Examine get_mnist() to provide your data in the correct format and structure. Basically you need to return a training set (x_train) a test set (x_test) and the belonging labels (y_test) for testing. Note that the images should be normalized as well
